README.md

```diff
@@ -38,6 +38,14 @@ async fn main() {
 }
 ```
 
+### Features
+
+OpenSSL Signature Validation is now optional, to use it
+
+``` features=["openssl_verifier]  ```
+
+By default, ```dummy_verifier``` implementation is selected if no custom verification implementation provided
```

Comment:
I would put this first in the paragraph. And then say, *In order to enable OpenSSL ...*

Reply:
done